dashboard.py

```
@@ -2187,8 +2194,11 @@ def _vllm_metric_cell(pct, better, similar):
     # ── Per-Accelerator Health Cards ────────────────────────────────
     st.markdown("### Performance by Accelerator")
     st.caption(
-        "Per-accelerator rollup averaged across all models and profiles — "
-        "status is determined by worst regression across all models."
+        "Per-accelerator rollup comparing the current release to the previous one. "
+        "**Avg Tput** is the mean throughput change across all combinations "
+        "(model × accelerator × profile). "
```

Ruff (0.15.13)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)